server/service/osquery_utils/queries.go

```diff
@@ -829,12 +829,31 @@ FROM cached_users CROSS JOIN atom_packages USING (uid);
 	DirectIngestFunc: directIngestSoftware,
 }
 
+var softwareChrome = DetailQuery{
+	Query: `SELECT
+  name AS name,
+  version AS version,
+  'Browser plugin (Chrome)' AS type,
+  'chrome_extensions' AS source,
+  '' AS vendor
+FROM chrome_extensions`,
+	Platforms:        []string{"chrome"},
+	DirectIngestFunc: directIngestSoftware,
+}
+
 var usersQuery = DetailQuery{
 	// Note we use the cached_groups CTE (`WITH` clause) here to suggest to SQLite that it generate
 	// the `groups` table only once. Without doing this, on some Windows systems (Domain Controllers)
 	// with many user accounts and groups, this query could be very expensive as the `groups` table
 	// was generated once for each user.
 	Query:            usersQueryStr,
+	Platforms:        []string{"linux", "macos", "windows"},
```

Comment:
Ah, this should actually be

```suggestion
	Platforms:        []string{"linux", "darwin", "windows"},
```

That explains at least one of the test failures.

Reply:
Thank you unit tests!